lib/connection/messages.coffee

```diff
@@ -75,3 +76,19 @@ module.exports =
       else
         ""
       dismissable: true
+
+  connectExternal: ->
+    tcp.listen().then (port) ->
+      msg = atom.notifications.addInfo "Connect an external process",
+        detail: """
+        To connect a Julia process running in the terminal,
+        run the command:
```

Comment:
Maybe mention that the current process needs to be stopped first? Or re-add the `direquire` stuff below?